docs/development/deploying-locally/index.md

```diff
@@ -42,25 +42,7 @@ $ firefox https://localhost:30443
 
 ## For developers
 
-If you are a developer and would like to work with REANA cluster source code, you can proceed as follows.
-
-[Fork the REANA main repository](https://github.com/reanahub/reana/fork) and clone it to install the `reana-dev` helper script:
-
-```console
-$ git clone git clone git@github.com:johndoe/reana
-$ cd reana
-$ virtualenv ~/.virtualenvs/reana
-$ source ~/.virtualenvs/reana/bin/activate
-(reana) $ pip install . --upgrade
-```
-
-Fork and clone all REANA cluster components repositories:
-
-```console
-(reana) $ reana-dev git-fork --help
-(reana) $ eval "$(reana-dev git-fork -c ALL)"
-(reana) $ reana-dev git-clone -c ALL -u johndoe
-```
+If you are a developer and would like to work with REANA source code, you can proceed as follows.
```

Comment:
Please keep the the word "cluster" in the introductory phrase, because theoretically one could work with the client without touching the cluster. Even though it is not very probable :)

Reply:
Ok, added back